examples/test_krec_save.py

```diff
@@ -10,20 +12,23 @@
     -v
 """
 
-import krec
 import argparse
-import numpy as np
-from pathlib import Path
-import uuid
-import time
 import logging
+import sys
+import time
+import uuid
+from typing import Dict, Tuple
```

Comment:
we are using python 3.11 so you can just use `dict` and `tuple` directly, no need to import these